docs/analyzer-rules/AZFW0008.md

```diff
@@ -0,0 +1,23 @@
+# AZFW0008: Multiple HTTP response binding types for Azure Function
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0008|
+| **Category** |[FunctionMetadataGeneration]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when there are multiple HTTP response binding types associated with an Azure Function.
+
+## Rule description
+
+Each Azure Function can have only one HTTP response binding type.
+
+## How to fix violations
+
```

Comment:
Feel free to add sample code snippets as you find appropriate. Ex: https://dotnet-worker-rules.azurewebsites.net/rules?ruleid=AZFW0002